modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java

```diff
@@ -490,29 +490,34 @@ public void deployObjectDefinition(ObjectDefinition objectDefinition) {
 	}
 
 	@Override
-	public ObjectDefinition enableAccountEntryRestricted(
-			ObjectRelationship objectRelationship)
+	public ObjectDefinition enableObjectRestricted(
```

Comment:
create another method to set account restriction for salesforce

Reply:
you could rename to enableSalesForceAccountEntryRestricted and enableAccountEntryRestricted